pages/howWeCalculate.js

```diff
@@ -1,41 +1,135 @@
 import { Flex, Text, UnorderedList, ListItem, Link } from "@chakra-ui/react";
 import Layout from "../components/Layout/Layout";
 import TextBlock from "../components/TextBlock/TextBlock";
+import CalculationFormula from "../public/images/how-we-calculate/calculation-formula.svg";
 
 export default function howWeCalculate() {
   return (
     <Layout isText={true} maxContainerWidth="100%">
       <Flex
-        direction={["column", "row"]}
+        direction={["column-reverse", "row"]}
         px={["10px", "50px"]}
         py="48px"
         gap="50px"
       >
         <Flex width={["100%", "70%"]} direction="column">
+          <Link
+            fontWeight={700}
+            fontSize="16px"
+            pb={["20px", "30px"]}
+            href="/results"
+          >
+            Work Commute Survey Results / Carbon Emissions Calculating and
+            reporting
```

Comment:
I think we can make the Back link title a little more terse:
```suggestion
            Back to Work Commute Survey Results
```
and maybe add the 'Back' Arrow icon (that we use in the survey forms) along with title, to make it more apparent that this is a clickable title?

![image](https://user-images.githubusercontent.com/4408259/181917393-78ef0ee4-7b6c-425b-b121-55df9a39166c.png)

Reply:
We can bring it up to design team on Tuesday if you like, I implemented the current figma design. 👍